src/Equinox/StreamId.fs

```diff
@@ -42,8 +42,8 @@ module StreamId =
 
 namespace Equinox
 
-/// Helpers for composing and rendering StreamId values
-type StreamId =
+[<AbstractClass; Sealed>]
+type StreamId private () =
```

Comment:
This should be public for C# consumption?

Reply:
That's just making it a real static class (so piping to it does not give weird syntax errors etc), i.e. the private is referring to the constructor that nobody will call

Reply:
And you're not fooling me with your concern for C#